demo-site/templates/origin/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:16-alpine as builder
+ARG ENFORCER_NAME
+
+WORKDIR /tmp/sample-sites
+
+COPY shared_config.json .
+COPY scripts scripts
+COPY utils utils
+COPY templates/static_files templates/static_files
+COPY servers/${ENFORCER_NAME} servers/${ENFORCER_NAME}
+
+RUN node scripts/create_static_files.js && node scripts/create_px_configs.js
```

Comment:
We called these functions to create static files and generate px configs because there were a bunch of different enforcers that needed the same static files and configurations. If we're moving all the sample sites into each individual enforcer repo, then we can simplify this.